src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -1778,11 +1778,11 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
 
     It takes three arguments:
 
-    * Attribute name
-    * Maximum value of the sample range. Any outliers will appear in the final bucket.
-    * Total number of buckets
+      * Attribute name
+      * Maximum value of the sample range. Any outliers will appear in the final bucket.
```

Comment:
Not sure there's anything to be done about this, but having the period at the end of this line while periods are absent from the others is inconsistent.